cli/search/search.go

```diff
@@ -25,6 +25,12 @@ const (
 	SearchAll
 )
 
+// defaultDirPermissions is rights used to create folders.
+// 0755 - drwxr-xr-x
+// We need to give permission for all to execute
```

Comment:
Who are "we"?
"drwxr-xr-x" - is a sufficient description.

Reply:
fixed